pkg/scheduler/factory/plugins.go

```diff
@@ -369,7 +369,14 @@ func buildScoringFunctionShapeFromRequestedToCapacityRatioArguments(arguments *s
 	if err != nil {
 		klog.Fatalf("invalid RequestedToCapacityRatioPriority arguments: %s", err.Error())
 	}
-	return shape
+	resourceToWeightMap := make(priorities.ResourceToWeightMap, 0)
```

Comment:
as we discussed offline, we should set the default value here in case arguments.ResourceToWeightMap is nil.